### DISCUSSION TITLE
Easy stackalloc support

### DISCUSSION THREAD

**Discussion by Xyncgas** [fsharp/fslang-suggestions#1238] [Library]
Add a keyword or CE or something to create a span / IsByRefLike struct that let me yield also ByRefLike struct inside, let me pass those around to other function, then further processing the stuffs creating more ByRef stuff by giving Seq/List/Array/Map/Set modules ```toSpan``` function

**Comment by vzarytovskii** [comment]
Please, create a language suggestion (or find existing one).
Thanks.
